src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -471,6 +472,45 @@ struct FaceCoordinatesCompute : FaceCoordinates<Dim, Frame, Centered>,
 };
 /// @}
 
+/// @{
+/*!
+ * \brief The value of the scalar field and its time derivative on element faces
```

Comment:
Can you add in the `\details` that if an element isn't on the face of the worldtube boundary that this will return a `nullopt`?